Mathlib/LinearAlgebra/Matrix/Zpow.lean

```diff
@@ -285,19 +287,23 @@ theorem zpow_mul' (A : M) (h : IsUnit A.det) (m n : ℤ) : A ^ (m * n) = (A ^ n)
   rw [mul_comm, zpow_mul _ h]
 #align matrix.zpow_mul' Matrix.zpow_mul'
 
+/-
+porting note: TODO: is this still needed, or should just the norm_cast be removed?
```

Comment:
Lemmas should almost never be removed during porting. Why does `norm_cast` complain?

Reply:
on the left hand side, `u ^ n` was interpreted in the wrong way, I think it should be `(u : M) ^ n`. Now it works